.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing to GPT Engineer
+
+By participating in this project, you agree to abide by the [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
I am aware that we don't have a `CODE_OF_CONDUCT.md` file yet. 
See [#48](https://github.com/AntonOsika/gpt-engineer/pull/48#issuecomment-1593916419)